package/gluon-web-model/files/lib/gluon/web/view/model/form.html

```diff
@@ -1,4 +1,4 @@
-<form method="post" enctype="multipart/form-data" action="<%|url(request)%>">
+<form method="post" enctype="multipart/form-data" action="<%|url(request)%>" data-update="reset">
```

Comment:
Where can I find documentation on `data-update`, I feel lost.

Reply:
We really need some documentation of the gluon-web{,-model} internals...

All gluon-web-model form elements specify a list of "interesting" events using the data-update attribute - usually containing the "change" event. Each listed event is bound to the *update* function in the global initialization code in `gluon-web-model.js`, which is responsible for updating the visibility of dependent elements.